content/docs/instrumenting/standard_exports.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Standard Exports
+sort_rank: 7
+---
+
+# Standard Exports
+
+This document covers what exports and collectors Prometheus client libraries
+should offer, with the aim of consistency across libraries, making the easy use
+cases easy and making it easier to provide tooling and consoles for common use
+cases.
+
+In addition, client libraries are ENCOURAGED to also offer whatever makes sense
+in terms of metrics for their language’s runtime (e.g. Garbage collection
+stats).
+
+These exports should have the prefix `process_`. If a language or runtime
+doesn't expose one of the variables it'd just not export it. All memory values
+in bytes, all times in unixtime/seconds.
+
+## Should
+
+| Metric name                     | Description                                            | Format  |
```

Comment:
I'd replace `Description` by `Help string` and make the help string given here mandatory. (There is still some stalled work lying around to unify the help strings. We can as well do it right now. But that requires to check where we have already consensus, and where we have deviations.)